### ISSUE TITLE
Hover API URL is incorrect

### ISSUE THREAD

**Issue by pro-sumer** [AnalogJ/lexicon#716]
The [URL](https://hoverapi.docs.apiary.io/) to the Hover API is incorrect.

Although that document describes an API and contains the word "Hover" multiple times, it does not list the unofficial API commands [discovered by Dan Krause](https://gist.github.com/dankrause/5585907). On top of that, it does not describe any DNS related requests at all, but instead a lot of requests for services that are not offered by [Hover](https://www.hover.com) (the domain registrar).

I suggest to remove this misleading URL from the GitHub home page for this project.

**Comment by adferrand** [comment]
So you think it would be relevant to not provide any URL at all?

**Comment by pro-sumer** [comment]
Since there apparently is no (un)official API documentation, either:
* don't provide a URL
* link to the Hover homepage

**Comment by lksoft** [comment]
I agree. The URL provided seems to not be related to the actual Hover service.

**Comment by jopeg** [comment]
I concur, I actually contacted hover to ask about this and was informed that they are aware of unofficial APIs, but that link is 100% not one of them. It is very misleading.

**Comment by pro-sumer** [comment]
What was the resolution? (If any?)

**Comment by Sudrien** [comment]
Updated URLs for hover, and TOTP secret support

https://github.com/AnalogJ/lexicon/issues/1602#issuecomment-1720377771 has my notes for installing the testing branch with pip - if you want to get in feedback before the next release

**Comment by pro-sumer** [comment]
#1718 does not change README.rst, so won’t the Hover API documentation link in there still be wrong then?

**Comment by adferrand** [comment]
You are right. I reopen the issue.
